CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing to Auth0 projects
+
+A big welcome and thank you for considering contributing to the Auth0 projects. It’s people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you agree to respect the time of the developers managing and developing these open source projects. In return, we will reciprocate that respect by addressing your issue.
+
+### Quicklinks
+
+* [Code of Conduct](#code-of-conduct)
+* [Getting Started](#getting-started)
+    * [Making Changes](#making-changes)
+    * [Opening Issues](#opening-issues)
+* [Getting in Touch](#getting-in-touch)
+    * [Got a question or problem?](#got-a-question-or-problem?)
+    * [Vulnerability Reporting](#vulnerability-reporting)
+
+## Code of Conduct
+
+By participating and contributing to this project, you are expected to uphold our [Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).
+
+## Getting Started
+
+### Making Changes
+
+When contributing to a repository, the first step is to open an issue in that repository to discuss the change you wish to make before making them.
```

Comment:
Do we need this? Do we want to recommend opening Github issue as well as support ticket?

Reply:
I removed this.